src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
```

Comment:
This method is being called from the finalizer.
It's members are not consider reachable either way as soon as the finalizer is done, and will be reclaimed (if no other objects reference them) on the next GC round in that generation.